stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/Page.java

```diff
@@ -147,6 +150,26 @@ public void clearDeviceOrientationOverride(JsonRpcPeer peer, JSONObject params)
   }
 
   @ChromeDevtoolsMethod
+  public void startScreencast(final JsonRpcPeer peer, JSONObject params) {
+    final StartScreencastRequest request = mObjectMapper.convertValue(
+        params, StartScreencastRequest.class);
+    mScreencastDispatcher = new ScreencastDispatcher();
```

Comment:
We need to check if `mScreencastDispatcher` is non-null and not overwrite it with a new reference.